### DISCUSSION TITLE
文字显示不正确

### DISCUSSION THREAD

**Discussion by gskrpb** [GuguTown/Wiki#3] [Q&A] [answered]
问题页面：https://gugutown.github.io/Wiki/article/calc_README.html
具体问题：“敌方格式”下的大段文字没有正确显示（然而在Github页面上能正确显示，怪哦）

**Comment by HazukiKaguya** [comment, marked as answer]
㊙，虽然不知道为啥但姑且是修好了

**Reply by HazukiKaguya** [reply]
顺便这种肯定是有问题的可以直接发issue（
